src/components/Breadcrumbs/BreadcrumbItem.tsx

```diff
@@ -37,20 +37,23 @@ const BreadcrumbItem: React.FC<BreadcrumbItemProps> = ({
   to = undefined,
   href = undefined,
   ...props
-}) =>
-  isSelected ? (
-    <Text
-      aria-current="page"
-      size={TextSizes.md}
-      variant={TextEnums.TextVariants.secondary}
-    >
-      {children}
-    </Text>
-  ) : (
-    <BreadcrumbLink href={href} to={to} {...props}>
-      {children}
-    </BreadcrumbLink>
-  );
+}) => (
+  <li>
```

Comment:
I'm thinking if we need some style reset for the platform. I remember there were issues with list styles before (rendered dots...). Did you test it?

Reply:
I will take a look at it

Reply:
yep, styles are broken, fixing it